docs/README.md

```diff
@@ -6,3 +6,4 @@
 -   [PdfClient](./pdf-client.md)
 -   [TeamMembersClient](./team-members-client.md)
 -   [JobsClient](./jobs-client.md)
+-   [Models](./models.md)
```

Comment:
Could we please create a file for each class and maybe have a new sub-heading for `Models` and put all the Client classes under a sub-heading `Clients`